app/index.js

```diff
@@ -139,7 +139,7 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       backgroundColor: toElectronBackgroundColor(cfg.backgroundColor || '#000'),
       // we want to go frameless on windows and linux
       frame: process.platform === 'darwin',
-      transparent: true,
+      transparent: process.platform === 'darwin',
```

Comment:
Will this cause an issue for people using Linux?

Reply:
Hi, I accidentally enabled transparency on my hyper. This causes the windows to fail to snap into fancy areas as before. I've tried turning it off but I cannot find the index.js file anywhere. I tried also uninstalling it but it somehow finds the old configurations after I re-install it.